src/main_app/tasks/utils/tasks_utils.py

```
@@ -95,3 +95,11 @@ def make_results_summary(
         "upload_result": upload_result,
         "main_title": main_title,
     }
+
+
+__all__= [
+    "json_save",
+    "commons_link",
+    "save_files_stats",
+    "make_results_summary",
+]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)